src/components/MemberListSummary/MemberListSummary.tsx

```diff
@@ -0,0 +1,131 @@
+import {
+  Avatar,
+  CardContent,
+  Grid,
+  makeStyles,
+  Typography,
+} from '@material-ui/core';
+import { Group } from '@material-ui/icons';
+import { AvatarGroup, Skeleton } from '@material-ui/lab';
+import { To } from 'history';
+import { FC } from 'react';
+import * as React from 'react';
+import { HugeIcon } from '../Icons';
+import { CardActionAreaLink } from '../Routing';
+
+const useStyles = makeStyles(({ spacing }) => ({
+  grid: {
+    marginBottom: spacing(2),
+  },
+  bottomContent: {
+    display: 'flex',
+  },
+  avatarGroup: {
+    marginRight: spacing(1),
+  },
+  skeletonText: {
+    marginLeft: spacing(1),
+  },
+}));
+
+export interface MemberSummaryItem {
+  // url to picture
+  picture?: string;
+  // will display these letters if picture url isn't given
+  avatarLetters?: string;
+  label: string;
+}
+
+export interface MemberListSummaryInterface {
+  memberSummaryItems?: MemberSummaryItem[];
+  to: To;
+}
+
+export const MAX_USER_TO_DISPLAY = 4;
+
+export const MemberListSummary: FC<MemberListSummaryInterface> = ({
+  memberSummaryItems,
+  to,
+}) => {
+  const classes = useStyles();
+  const membersToDisplay = memberSummaryItems?.slice(0, MAX_USER_TO_DISPLAY);
+  const amountOfNonDisplayedMembers = Math.max(
+    (memberSummaryItems?.length ?? 0) - MAX_USER_TO_DISPLAY,
+    0
+  );
+  const firstCoupleNamesString = membersToDisplay?.reduce(
+    (accumulator, member, index) => {
+      if (index === 0) {
+        return `${member.label}`;
+      } else {
+        return `${accumulator}, ${member.label}`;
+      }
+    },
+    ''
+  );
+  const createEmptyAvatars = () => {
+    const emptyAvatars = [];
+    for (let i = 0; i < MAX_USER_TO_DISPLAY; i++) {
+      emptyAvatars.push(<Skeleton variant="circle" width={36} height={36} />);
+    }
+    return emptyAvatars;
+  };
+
+  return (
+    <CardActionAreaLink to={to} external={false}>
+      <CardContent>
+        <Grid container spacing={4} className={classes.grid}>
+          <Grid item>
+            <HugeIcon icon={Group} />
+          </Grid>
+          <Grid item>
+            <Typography>Team Members</Typography>
```

Comment:
This needs to be passed in. Maybe `title`?

Reply:
I used title as the prop name for this